jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -1428,20 +1409,24 @@ private String findServerName()
             }
         }
 
+        if (_channel != null)
+        {
+            Connector connector = _channel.getConnector();
+            if (connector != null)
+            {
+                HostPort serverAuth = connector.getServerUriAuthority();
+                if (serverAuth != null)
+                {
+                    return formatAddrOrHost(serverAuth.getHost());
+                }
+            }
```

Comment:
This feels wrong to be diving down to the connector  here.  Would be better just to ask for `_channel.getServerAuthority()`